test/integration/all-events-consolidated.sh

```diff
@@ -103,6 +103,72 @@ make deploy-artifact \
   activate=false \
   force=true
 
+##################################################
+# Enable scheduled jobs for offline-snapshot
+##################################################
+
+make schedule-offline-snapshot-consolidated \
```

Comment:
The events for offline snapshot schedule should be consistent with other event like crxde, i.e. it should be called `enable-<something>` and `disable-<something>`.
So in this case, it should be `enable-scheduled-offline-snapshot-consolidated`. The state value shouldn't matter for the user, users don't need to know what the value of state should be, whether it be `enable` or `present` or `yes`, should be hidden from them.

Reply:
Done with the latest push.